src/cascadia/TerminalSettingsEditor/Appearances.xaml

```diff
@@ -23,6 +23,12 @@
                           x:DataType="local:EnumEntry">
                 <RadioButton Content="{x:Bind EnumName, Mode=OneWay}" />

```

Comment:
is the RadioButton version still used?

Reply:
This probably applies to the copied over version in the other files.